api/shared/last_operation.go

```diff
@@ -0,0 +1,10 @@
+package shared
```

Comment:
I like the changes, but can we talk about the package name?
From usage perspective `shared.State` or `shared.Resource` seems a bit weird, wdyt? The intention is to share that's right, but the actual property of the package is that it contains common types to be used from various consumers esp. `api`.
So what do you think about naming it `common` or `general`? The import reads nice and explains a bit:
`github.com/kyma-project/lifecycle-manager/api/common`
Just a thought, there maybe better names

Reply:
I use `common` in the first try, but the name `common` is too common even in the LM repo, we already have two package named `common`, I don't want to introduce one more. I know this name is not perfect, but compared to `general`, it's better, general is way too general.

The intention of `shared` is sharing this type, that's right. For example, `shared.State` is a state shared by kyma, manifest, module CR, `shared.LastOperations` is same purpose. `shared.Resource` is only for manifest CR atm, but it can be used in future types if needed.

Reply:
Alright if `common` is already taken too much, let's keep it 👍